code/game/objects/item.dm

```diff
@@ -992,3 +985,11 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 
 /obj/item/proc/gives_weather_protection()
 	return FALSE
+
+//#TODO: Stub implementation. Probably should be unified into /obj along with health..
+/obj/item/proc/take_damage(var/amount, var/damtype, var/silent = FALSE)
```

Comment:
This is at least a step closer to unification.

Reply:
Hopefully XD